From 46b14df152bb0dd47631c2ef5902f0a9b44ee7be Mon Sep 17 00:00:00 2001
From: Olly Betts <olly@survex.com>
Date: Fri, 22 Jun 2018 13:19:03 +1200
Subject: [PATCH] Improve wxTreeCtrl::SetItemState() docs

Make it clear that the state parameter can be an index into the state
image list, not just one of the special values listed.
---
 interface/wx/treectrl.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/interface/wx/treectrl.h b/interface/wx/treectrl.h
index c7738861..24616505 100644
--- a/interface/wx/treectrl.h
+++ b/interface/wx/treectrl.h
@@ -902,7 +902,8 @@ public:
                               wxTreeItemIcon which = wxTreeItemIcon_Normal);
 
     /**
-        Sets the specified item state. The value of @a state may be:
+        Sets the specified item state. The value of @a state may be an index
+        into the state image list, or one of the special values:
         - @c wxTREE_ITEMSTATE_NONE: to disable the item state (the state image will
             be not displayed).
         - @c wxTREE_ITEMSTATE_NEXT: to set the next item state.
-- 
2.18.0.rc2

